pkgs/os-specific/linux/ebtables/default.nix

```diff
@@ -1,6 +1,17 @@
-{ stdenv, fetchurl }:
+{ lib, stdenv, fetchurl, enableDebug ? false, enableStatic ? false }:
 
-stdenv.mkDerivation rec {
+let staticBuildOpts = {
```

Comment:
I think it would be more clear to put these inside of mkDerivation. For instance:

```
  buildFlags = if enableStatic then "static" else null;
  installPhase = if enableStatic then ''
    ...
  '' else null;
```